src/css/base/_reset.scss

```diff
@@ -144,3 +145,112 @@ cite {
     font-size: 36px;
   }
 }
+
+.dark {
+  body {
+    background-color: var(--dark-surface-base);
+    background-image: linear-gradient(to top left, rgba(32, 139, 238, 0.06), rgba(32, 139, 238, 0.12));
+    color: var(--dark-typography-paragraph);
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: var(--dark-typography-title);
```

Comment:
Consider creating a new CSS variable to define colors in both default and dark mode in `_colors.scss`, rather than directly selecting ".dark h1, h2, h3,...". Applies to other elements (e.g., p, body, code).